src/python/tensorflow_cloud/core/tests/unit/validate_test.py

```diff
@@ -13,29 +13,23 @@
 # limitations under the License.
 """Tests for the validation module."""
 
+import mock
 import os
-import pytest
```

Comment:
FWIW, dependency on pytest itself isn't a bad thing, because it can do stuff that unittest doesn't do. 

The gist of the issue was that the previous code 'required' pytest also as the test driver.

Now that you addressed this test method can be called as authentic python program, problem solved.

Reply:
Makes sense, thank you Makoto!